src/Workspaces/SharedUtilitiesAndExtensions/Compiler/Core/Options/EditorConfig/EditorConfigStorageLocation.cs

```diff
@@ -46,9 +39,6 @@ private static Optional<string> ParseString(string str)
             return str.Replace("\\r", "\r").Replace("\\n", "\n");
         }
 
-        private static readonly Func<int, string> s_getInt32EditorConfigStringForValue = GetInt32EditorConfigStringForValue;
-        private static string GetInt32EditorConfigStringForValue(int value) => value.ToString().ToLowerInvariant();
-
         private static readonly Func<string, string> s_getStringEditorConfigStringForValue = GetStringEditorConfigStringForValue;
         private static string GetStringEditorConfigStringForValue(string value) => value.ToString().Replace("\r", "\\r").Replace("\n", "\\n");
```

Comment:
in `GetBoolCodeStyleOptionEditorConfigStringForValue` below (can't add a comment directly on it) - I think that all needs to call the editorconfig data object right to get the true or false right?  Same with `ParseStringCodeStyleOption` and `GetStringCodeStyleOptionEditorConfigStringForValue`